frontend/src/app/modules/header/header-item/header-item.component.html

```diff
@@ -16,8 +16,8 @@
         </nav>
 
         <button class="menu-team-btn" mat-button [matMenuTriggerFor]="menuTeam" *ngIf="teams.length; else newTeam">
-            <label>{{currentTeam?.name ?? ''}}</label>
-            <img src="assets/icons/angle-down.svg"  alt="angle-down"/>
+            <label class="current-team">{{ currentTeam?.name ?? '' }}</label>
```

Comment:
```suggestion
            <label>{{ currentTeam?.name ?? '' }}</label>
```
`current-team` class is not used anywhere